### PR TITLE
Legacy conversion of Password Managers

### PR DESCRIPTION
**Pull request by dngray**
Closes: https://github.com/privacyguides/privacyguides.org/issues/10
Closes: https://github.com/privacyguides/privacyguides.org/discussions/47
Closes: https://github.com/privacyguides/privacyguides.org/pull/17

Depends on: https://github.com/privacyguides/privacyguides.org/pull/17

### CODE REVIEW

**Comment by dngray** [comment]
This one is pretty much ready. I think @tommytran732 wanted to write something about how the F-Droid version of Bitwarden and Google Play edition compares.

File: _data/software/password-manager-servers/2_psono_server.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+title: Psono Server
+type: Recommendation
+logo: /assets/img/password-management/psono.svg
+description: |
+  Psono provides [extensive documentation](https://doc.psono.com/) for their product. The [web-client](https://doc.psono.com/admin/installation/install-webclient.html#installation-with-docker) for Psono can be self hosted, alternatively you can choose the the full [Community Edition](https://doc.psono.com/admin/installation/install-server-ce.html#preamble) or the [Enterprise Edition](https://doc.psono.com/admin/installation/install-server-ee.html) with additional features.
```
Comment (elitejake) [inline _data/software/password-manager-servers/2_psono_server.yml:5]: The `#preamble` in community edition link is not necessary, right?

File: _data/software/password-manager-servers/1_vaultwarden.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+title: Vaultwarden
+type: Recommendation
+logo: /assets/img/password-management/vaultwarden.svg
+logo_dark: /assets/img/password-management/vaultwarden-dark.svg
+description: |
+  Vaultwarden is an alternative implementation of the Bitwarden server API written in Rust and compatible with upstream Bitwarden clients*, perfect for self-hosted deployment where running the official resource-heavy service might not be ideal.
```
Comment (elitejake) [inline _data/software/password-manager-servers/1_vaultwarden.yml:6]: The `*` in `Bitwarden clients*,` is not addressed.

File: collections/_evergreen/password-management.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+title: Password Managers
+mathjax: false
+description: |
+  Stay safe and secure online with an encrypted and open-source password manager.
+---
+
+## Password best practices
+- Always use unique passwords. Don't make yourself a victim of "[credential stuffing](https://en.wikipedia.org/wiki/Credential_stuffing)".
+- Store an exported backup of your passwords in an [encrypted container](/software/file-encryption/) on another storage device. This can be useful if something happens to your device or the service you are using.
+- Store [Time-based one-time password (TOTP)](https://en.wikipedia.org/wiki/Time-based_one-time_password) tokens in a separate [TOTP app](/password-management#time-based-one-time-password-managers) and not your password manager. TOTP codes are generated from a "[shared secret](https://en.wikipedia.org/wiki/Time-based_one-time_password#Security)". If the secret is obtained by an adversary they can generate TOTP values. Typically mobile platforms have better app isolation and secure better methods for storing sensitive credentials.
```
Comment (elitejake) [inline collections/_evergreen/password-management.md:12]: The TOTP app link (`/password-management#time-based-one-time-password-managers`) does not exist.

File: _data/software/password-manager-minimal/2_gopass.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+title: gopass
+type: Recommendation
+logo: /assets/img/password-management/gopass.png
```
Comment (elitejake) [inline _data/software/password-manager-minimal/2_gopass.yml:3]: They have an svg logo [here](https://github.com/gopasspw/gopass/blob/master/docs/logo.svg).

**Review by elitejake — CHANGES_REQUESTED** [review]

File: _data/software/password-manager-cloud/1_bitwarden.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+title: Bitwarden
+type: Recommendation
+logo: /assets/img/password-management/bitwarden.svg
+description: |
+  **Bitwarden** is a free and open-source password manager. It aims to solve password management problems for individuals, teams, and business organizations. Bitwarden is among the easiest and safest solutions to store all of your logins and passwords while conveniently keeping them synced between all of your devices. If you don't want to use the Bitwarden cloud, you can easily host your own Bitwarden server.
+
+  ## Notes
```
[the remark this answers is not in the document]
Comment (elitejake) [reply _data/software/password-manager-cloud/1_bitwarden.yml:7]: Wait, is this section left blank for @tommytran732 to add F-Droid vs Play Store comparisons?

[the remark this answers is not in the document]
Comment (dngray) [reply _data/software/password-manager-cloud/1_bitwarden.yml:7]: Yes.

File: _data/software/password-manager-local/2_keepassdx.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+title: KeepassDX
+type: Recommendation
+logo: /assets/img/password-management/keepassdx.svg
+description: |
+  **KeepassDX** is a modern password manager which by default stores its database locally and not in the cloud. The [Contributor Pro](https://play.google.com/store/apps/details?id=com.kunzisoft.keepass.pro)  package adds some extra features to the app.
+
+  For more details we recommend looking at their [FAQ](https://github.com/Kunzisoft/KeePassDX/wiki/FAQ).
```
[the remark this answers is not in the document]
Comment (dngray) [reply _data/software/password-manager-local/2_keepassdx.yml]: :+1:

File: collections/_evergreen/password-management.md (hunk 3 above)
Comment (dngray) [reply collections/_evergreen/password-management.md:12]: Correct, https://github.com/privacyguides/privacyguides.org/pull/17 must be completed and merged first.

File: _data/software/password-manager-servers/2_psono_server.yml (hunk 1 above)
Comment (dngray) [reply _data/software/password-manager-servers/2_psono_server.yml:5]: No, that was an error.

File: _data/software/password-manager-servers/1_vaultwarden.yml (hunk 2 above)
Comment (dngray) [reply _data/software/password-manager-servers/1_vaultwarden.yml:6]: Well spotted.

**Pull request merged**